src/network/clientpackethandler.cpp

```diff
@@ -435,6 +435,9 @@ void Client::handleCommand_ChatMessage(NetworkPacket *pkt)
 
 	chatMessage->type = (ChatMessageType) message_type;
 
+	// log the chat message
+	actionstream << "[Chat] " << wide_to_utf8(chatMessage->message) << std::endl;
```

Comment:
The tag string should be `"CHAT: "` for consistency, see: https://github.com/minetest/minetest/blob/fbec168e91026b70fd989e7f46bea7140328076e/src/server.cpp#L3115.